src/components/results/results-view-selector/index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+
+import './style.css';
+
+export const TREE_VIEW = 'tree_view';
+export const JSON_VIEW = 'json_view';
+
+
+const ResultsViewSelector = ( { view, onViewChange = () => {} } ) => (
+	<div className={ 'results-view-selector__button-selector-container' }>
+		<button
+			onClick={ () => onViewChange( TREE_VIEW ) }
+			className={ `results-view-selector__button-selector ${ view === TREE_VIEW ? 'selected' : '' }` }
```

Comment:
it's fine using these inline ternaries, but one thing I noticed now that I didn't before is that `classnames` is in use already in this project, even in files in this PR. so it's okay to leave it, it's okay to change it (there aren't any dependency implications or new code styles being introduced)

```ts
className={ classnames( 'results-view-selector__button-selector', {
	selected: view === TREE_VIEW
} ) }
```